### ISSUE TITLE
No spacing between webform elements

### ISSUE THREAD

**Issue by brharp** [ccswbs/hjckrrh#492]
There is no space between webform elements.

Bootstrap provides a class 'form-group' that can be applied to the label/form wrapper to provide the proper spacing. E.g.
```
<div class="form-group form-item webform-component webform-component-textfield webform-component--first-name">
  <label for="edit-submitted-first-name">First Name <span class="form-required" title="This field is required.">(required)</span></label>
  <input required="required" class="form-control form-text required" id="edit-submitted-first-name" name="submitted[first_name]" value="" size="60" maxlength="128" type="text">
</div>
```
(note addition of the "form-group" class to the div)

It should be possible to add this class to the wrapper by overriding `theme_webform_element`.

**Comment by abanuog** [comment]
https://www.drupal.org/node/2712217 - perhaps we can use the patch, and the bootstrap defaults will get applied.

**Comment by brharp** [comment]
Fixed in 7.8.0
